Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/type/CoronaTest.kt

```diff
@@ -65,3 +69,18 @@ interface CoronaTest : Recyclable {
 
 typealias RegistrationToken = String
 typealias TestIdentifier = String
+
+/**
+ * Tells if a [CoronaTest] age is older than the duration defined by [CoronaTestConfig]
+ */
+fun CoronaTest.didPassConfigDuration(
+    coronaTestConfig: CoronaTestConfig,
```

Comment:
can you rename to make it more clear what it means? eg.
allowsLowRiskCardToShow()

Reply:
it is not only low ,renamed to `shouldShowRiskCard`